### PR TITLE
Add default prompt value for user prompt WebDriver tests

### PR DESCRIPTION
**Pull request by jimevans**
When the JavaScript `prompt()` function is called without a default value,
some browsers (like IE) return "undefined" for the default prompt value if
the argument is omitted. This commit refactors the user prompt handler
tests so that prompt() is called with an appropriate value for the default
value argument, providing a consistent behavior across browsers.

### CODE REVIEW

**Comment by andreastt** [comment]
I think @whimboo should have a look at this.

**Comment by whimboo** [comment]
Hi @jimevans. Thank you for the PR. I had a quick glance over it and two questions came up for me:

1) Which versions of IE is your driver still supporting? As I read on the cross-browser [MDN documentation for `prompt`](https://developer.mozilla.org/en-US/docs/Web/API/Window/prompt) this specific `default value` behavior is only necessary for IE7 and IE8. So I assume those are still on your compatibility list?

2) I don't think that we should expose the default value all the way down to the tests. Also because it adds a huge amount of boiler-plate. Instead why can't we just hide it's usage inside of the `create_dialog` fixture? I feel it's a proper place, and we can determine it by the `dialog_type` argument.

**Comment by jimevans** [comment]
@whimboo

1. The docs are wrong. I tested this against IE11, and the same behavior happens even outside the WPT test harness. 

2. That’s a perfectly fine approach. I’ll refactor to make that work.

File: webdriver/tests/support/fixtures.py
Comment (whimboo) [inline webdriver/tests/support/fixtures.py]: Please note that IE needs that.

File: webdriver/tests/support/fixtures.py
Comment (whimboo) [inline webdriver/tests/support/fixtures.py]: Mind, completely moving this into the script code so that we can get rid of the call to `format()`?

**Review by whimboo — CHANGES_REQUESTED** [review]

File: webdriver/tests/support/fixtures.py
Comment (jugglinmike) [inline webdriver/tests/support/fixtures.py]: Kudos for resisting "(i.e. IE)" :P

Comment (jimevans) [reply webdriver/tests/support/fixtures.py]: @jugglinmike True confession time, I very nearly did "(e.g., IE)".

**Review by whimboo — APPROVED** [review]
What a difference to the first version. Thanks a lot for this addition @jimevans! Would you mind to update the MDN article?

**Comment by whimboo** [comment]
Oh, and please don't forget to squash before merge. Thanks.

**Comment by jgraham** [comment]
wdspec timeouts.

**Pull request merged**
